lambdas/db-migration/src/migrations/20201014105054_create_pdrs_table.ts

```diff
@@ -22,8 +19,36 @@ export const up = async (knex: Knex): Promise<void> =>
     table
       .integer('executionCumulusId')
       .references('cumulusId')
-      .inTable('executions')
+      .inTable('executions');
+    table
+      .enum('status', ['running', 'failed', 'completed'])
```

Comment:
Did we make a decision about `useNative`? I suppose it's fine to add later if there seems to be a compelling reason

Reply:
I opted not to use it because:

1) We're not reusing it in multiple locations 
2) It's not valuable for our interface to programmatically query *against the database* for valid enum fields 
3) The table configuration/reading what's going on isn't straightforward, and/or the enum syntax isn't as portable

and most critically 

4) I was unable to find a performance related reason to use it.